src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -307,8 +306,6 @@ public static void TCP_ServiceCertExpired_Throw_MessageSecurityException()
             {
                 Assert.True(false, string.Format("Expected type SecurityNegotiationException, Actual: {0}", exceptionType));
             }
-            string exceptionMessage = exception.Message;
-            Assert.True(exceptionMessage.Contains(Endpoints.Tcp_ExpiredServerCertResource_HostName), string.Format("Expected message contains {0}, actual message: {1}", Endpoints.Tcp_ExpiredServerCertResource_HostName, exceptionMessage));
```

Comment:
We expect the content of exception messages to not be the same between NetNative and NetCore. In this case we deliberately use an expired certificate and we get the expected SecurityNegotiationException so an additional check of the message content for the service host name is unnecessary.